### PR TITLE
Mark the aggregation 'Custom Expression' button as an action

### LINKED ISSUE #41403: Inconsistent styling in aggregation dropdown
### Describe the bug

"Custom Expression" button in aggregations dropdown is inconsistent with filters.

### To Reproduce

1. New Question → Sample data set → Orders
2. Add an aggregation 
3. Scroll down to the Custom Expression item

As you can see it is missing a border and a chevron like the filter does.

image
Screenshot 2024-04-15 at 12 42 09

### Expected behavior

The styles should be consistent

### Logs

_No response_

### Information about your Metabase installation

```JSON
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.20.1+1",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.20.1",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.20.1+1",
    "os.name": "Mac OS X",
    "os.version": "14.2.1",
    "user.language": "en",
    "user.timezone": "Europe/Amsterdam"
  },
  "metabase-info": {
    "databases": [
      "h2",
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "h2",
    "application-database-details": {
      "database": {
        "name": "H2",
        "version": "2.1.214 (2022-06-13)"
      },
      "jdbc-driver": {
        "name": "H2 JDBC Driver",
        "version": "2.1.214 (2022-06-13)"
      }
    },
    "run-mode": "dev",
    "version": {
      "date": "2024-04-11",
      "src_hash": "6c6fb78ac9f693ca14ae80f38f0bf9b3890c2772",
      "tag": "v1.48.1-SNAPSHOT",
      "hash": "9797bb1"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

### Severity

P3

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by romeovs**
Closes https://github.com/metabase/metabase/issues/41403

### Description

Marks the aggragation  'Custom Expression' button as an action to make the styling consistent.

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New Question → Sample data set → Orders
2. Add an aggregation
3. Scroll down to the Custom Expression item

The button should have a border and a chevron.

### Demo

Screenshot 2024-04-15 at 12 45 12

### CODE REVIEW

File: frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx [hunk 1]
```diff
@@ -122,6 +123,7 @@ export function AggregationPicker({
         name: t`Custom Expression`,
         items: [],
         icon: "sum",
+        type: "action",
```
Comment (kamilmielnik) [inline frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx:126]: Custom expression button now has green text color

image

**Review by kamilmielnik — CHANGES_REQUESTED** [review]

File: frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx (hunk 1 above)
Comment (romeovs) [reply frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx:126]: This makes it consistent with the other buttons. But I agree that it is a bit much compared to the other buttons, which have a more toned down active color.

@AlessioLaiso should we remove the color for buttons in the dropdown altogether?

Comment (AlessioLaiso) [reply frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx:126]: We explored adding color along these lines but decided against it. This green on white isn't great for accessibility either. TL;DR here: Divider and chevron are a yes, green text is a no.

**Comment by romeovs** [comment]
Superseded by https://github.com/metabase/metabase/pull/41565

**Pull request closed without merging**